Cargo.toml

```diff
@@ -1,16 +1,3 @@
-[package]
-name = "bevy"
-version = "0.9.0-dev"
-edition = "2021"
-categories = ["game-engines", "graphics", "gui", "rendering"]
-description = "A refreshingly simple data-driven game engine and app framework"
-exclude = ["assets/", "tools/", ".github/", "crates/", "examples/wasm/assets/"]
-homepage = "https://bevyengine.org"
-keywords = ["game", "engine", "gamedev", "graphics", "bevy"]
-license = "MIT OR Apache-2.0"
-readme = "README.md"
-repository = "https://github.com/bevyengine/bevy"
-
 [workspace]
 exclude = ["benches", "crates/bevy_ecs_compile_fail_tests"]
```

Comment:
Since `benches` and `crates/bevy_ecs_compile_fail_tests` are not members of workspace they cannot use workspace inheritance.

Reply:
removed these two again 👍